spec/orders_seeds.sql

```diff
@@ -0,0 +1,6 @@
+TRUNCATE TABLE orders RESTART IDENTITY;
+
+
+INSERT INTO orders (order, customer_name, order_date) VALUES (123, 'Calum Wilmot', 1/1/21);
```

Comment:
Check the 'date' format (may be 2022-01-02) https://www.w3schools.com/sql/sql_dates.asp#:~:text=SQL%20Date%20Data%20Types&text=DATE%20%2D%20format%20YYYY%2DMM%2D,YEAR%20%2D%20format%20YYYY%20or%20YY